projects/igniteui-angular/src/lib/combo/combo.common.ts

```diff
@@ -1154,6 +1154,7 @@ export abstract class IgxComboBaseDirective extends DisplayDensityBase implement
 
     protected findMatch = (element: any): boolean => {
         const value = this.displayKey ? element[this.displayKey] : element;
+        if (!value) return false;
```

Comment:
Eslint is complaining because the body of the if needs to be wrapped in braces and it should be on a new line.